cms/templates/partials/program-description.html

```diff
@@ -25,7 +25,7 @@ <h2><div class="description">{{ page.heading }}</div></h2>
           {% for step in page.steps %}
           <li class="panel-title">
             <a data-toggle="collapse" data-target="#collapse-{{forloop.counter}}" aria-expanded="{% if forloop.counter == 1 %}true{% else %} false {%endif%}" aria-controls="collapse-{{forloop.counter}}" href="#" class="opener"><span> &gt; </span> {{ step.value.title }} </a>
-            <div id="collapse-{{forloop.counter}}" class="collapse {% if forloop.counter == 1 %} show {%endif%}" aria-labelledby="heading-{{forloop.counter}}" data-parent="#accordionSteps" class="slide">
+            <div id="collapse-{{forloop.counter}}" class="collapse {% if forloop.counter == 1 %} show {%endif%}" aria-labelledby="collapse-{{forloop.counter}}" data-parent="#accordionSteps" class="slide">
```

Comment:
This is incorrect. The label is inside the anchor tag. You need to add an ID on the anchor tag and use that within the `aria-labelledby` attribute.